### PR TITLE
Mobilefirstfinal

### PR DESCRIPTION
**Pull request by leloufadel**
In this pull request: we have created a contact form.
We make sure that:
Name field accepts up to 30 chars.
Email field accepts only emails.
Message field accepts up to 500 chars.
All fields have placeholder text.
All fields have the correct type.
A user cannot submit the form if one or more of the fields is empty.
Collect data by using the [Formspree service](https://formspree.io/html).
in addition to that we:
use Flexbox.
stick to the design as much as possible (e.g., font, colors, images, text, margins) using the figma
Implement the button interactions (enable, hover, etc.).

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -306,3 +306,99 @@ h2 {
...
+.content-1 {
+  background-color: #6070ff;
+  background-image: url(image/Contact1.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 779px;
+  width: 375px;
+  border-top-left-radius: 100px;
+  padding: 8px;
+}
```
Comment (didierganthier) [inline styles.css]: Well done overall designing this 💯 but the default display is being used here, which is `display: block;`, you should be using `Flexbox` to layout the elements, this gives you much more option and it makes it easy to handle the responsiveness, you can start using flexbox by adding this line :
```
display: flex;
```
You can learn more about Flexbox [here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) and [here](https://www.w3schools.com/css/css3_flexbox.asp) 🔍 🤓

Comment (didierganthier) [inline styles.css]: Well done overall designing this 💯 but the default display is being used here, which is `display: block;`, you should be using `Flexbox` to layout the elements, this gives you much more option and it makes it easy to handle the responsiveness, you can start using flexbox by adding this line :
```
display: flex;
```
You can learn more about Flexbox [here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) and [here](https://www.w3schools.com/css/css3_flexbox.asp) 🔍 🤓

File: index.html [hunk 2]
```diff
@@ -177,11 +177,35 @@ <h2>About Myself</h2>
...
+              <li>
+             
+                <input type="email" id="mail" placeholder="Email address" name="user_email" required/>
+              </li>
+              <li>
+               
+                <textarea id="msg" placeholder="write a message here..." name="user_message" maxlength = "500" required></textarea>
+              </li>
+          <li>
+            <button class="in-touch" type="button">Get in touch</button>
```
Comment (didierganthier) [inline index.html]: Nice looking form 😍 but unfortunately it's not being submitted 😢 why is that? The reason is simple, this button has a `type` of `button` which means the form doesn't behave with it. 💨  The submit button can be either an `input` element with type `submit` or a `button` element with type `submit` or with no `type` attribute 🔍  (note that it defaults to a `submit` type when no `type` attribute is used.) So you could either have this:
```
<button class="in-touch" type="submit">Get in touch</button>
```
or this 🤷🏽 
```
<button class="in-touch">Get in touch</button>
```
It's very much up to you 🤓

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi @leloufadel and @Danny-imo ,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Used Formspree ✔️ 

## Required Changes 🛠
- [ ] The README looks nice 💯 but the `MIT` link is broken for now, the reason is your README is looking for a file named `MIT` while your file is named `MIT.md`, kindly adjust that to fix that issue. 
- [ ] Last and least thing about your README 👀 to make it look more original, you can either remove the Microverse logo completely or replace it with yours. You can find some cool ones on [Flaticon](https://www.flaticon.com/) 🤓 

- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

File: styles.css (hunk 1 above)
Comment (leloufadel) [reply styles.css]: Thank you, @didierganthier, I used display: flex on the form.
 when I used display flex in content-1 class, everything mixed up, the right place in my code to place it is the "form".

Comment (huseyinbiyik) [reply styles.css]: Hey @leloufadel. As the previous reviewer requested, please refactor your contact form area with Flexbox. 👍  In the very next lessons, you will build the desktop version of the portfolio and the webpage needs to respond to different screen sizes and that's why Flexbox was invented.  You can check [this lesson](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) to learn more about Flexbox.
Screenshot 2023-04-04 at 8 59 55 PM

File: styles.css [hunk 3]
```diff
@@ -306,3 +306,99 @@ h2 {
...
+  background: #fff;
+  border-radius: 8px;
+}
+
+.in-touch:hover {
+  background-color: #6070ff;
+  color: #fff;
+  border-color: #fff;
+  cursor: pointer;
+}
```
Comment (huseyinbiyik) [inline styles.css]: [OPTIONAL] Kindly follow the design guideline on Figma for all the styling values. And fix your Hover and Active(Pressed) state of your buttons 👍 🔘 
Screenshot 2023-04-04 at 8 51 24 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi team,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
tenor

 ## Highlights
- Form submits the data 👍 
- Linter set up well and checks pass 🟢

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
⚠️ Unfortunately the previous review @didierganthier did not notice that you are following incorrect flow for your git. For this project, we need to follow the Github flow and every feature branch needs to be merged into the main/master branch. ⚠️ 

-  Please make sure your commit messages are meaningful and descriptive for your next commits. Use imperative tense and capitalize the first word. For example, 
👌"Add logo to navigation bar"
👌"Fix close button functionality on mobile navbar"
❌ "Added logo to navigation bar"
❌ "Added HTML" 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @leloufadel , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Green linters
✅ ... Good design

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**